CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+#Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other standard  method with the owners of this repository before making a change.
+
+## Pull Request process
+
+1.Clone the repository 
+2.Install all the needed dependencies 
+3.Get an issue assigned to you by the developers
+4.Raise a PR with the issue number and  a description of changes done  
+5.Request the reviewers to merge the issue for you 
+
+##Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+### Our Standards 
```

Comment:
isn't this part of code of conduct why should we mention this in our contribute.md